src/bundle.ts

```diff
@@ -209,7 +209,12 @@ function runWatch(
   watcher.on('event', (event) => {
     switch (event.code) {
       case 'ERROR': {
-        return onError(event.error)
+        try {
+          onError(event.error)
```

Comment:
This will double log the error, let's use console.error to to log `event.error` here when it's available

```js
case "ERROR":
   if (event.error) { console.error(event.error) }
   break

```